gluu-server/opt/gluu/jetty/oxauth/custom/scripts/sic/oob.py

```diff
@@ -130,14 +136,19 @@ def AuthenticateOutOfBand(self, requestParameters):
         if enteredCode == identity.getWorkingParameter("oobCode"):
             facesMessages.clear()
             updateNeeded = False
+            telemetry["result"] = "sucess"
```

Comment:
typo

Reply:
Good catch. Thanks.

https://github.com/sign-in-canada/Acceptance-Platform/commit/993a22dc7ba0631f88d4021b57789a0d13e6012a